tweeter.py

```diff
@@ -99,7 +99,25 @@ def stats():
 
     return stat_string_1 + stat_string_2 + stat_string_3 + stat_string_4
 
-#"Tweeter Stats - Number of adjectives: <num_adj>, Number of objects: <num_obj>, Number of antecedents: <num_ant>, Number of postcedents: <num_post>"
+# 
+
+@app.route("/reversedtweet")
+def reversedtweet():
+    adj_index = random.randint(0, len(adj_list) - 1)
+    obj_index = random.randint(0, len(obj_list) - 1)
+    ant_index = random.randint(0, len(antecedent_list) - 1)
+    post_index = random.randint(0, len(postcedent_list) - 1)
+    tweet_subj = "{} {}".format(adj_list[adj_index], obj_list[obj_index])
+    tweet = "{} {}".format(antecedent_list[ant_index].format(tweet_subj), postcedent_list[post_index])
+
+    reversedtweet  = ""
+
+    for x in range(len(tweet)):
```

Comment:
Reverse looping over the string is the canonical solution for this task. We can however remove the need for the `reverse_x` index by iterating over each character directly:
```
for c in tweet:
    reversed_tweet = c + reversed_tweet
```
Another option is traversing the string from both ends and reversing it in-place, something like:
```
for i in range(len(tweet)):
    j = len(tweet) - i - 1
    if i > j:
         break
    temp_char = tweet[i]
    tweet[i] = tweet[j]
    tweet[j] = temp_char
```
This method is more memory efficient since there is at most one extra character worth of memory occupied at any time as opposed to an entire string of the same size.

There's even more efficient ways that involve bit manipulation but what we should do in python is use either the `reversed()` builtin or slicing (this is the fastest method).
`reversed_tweet = tweet[::-1]`

We should use the string slicing solution but record the in-place solution in a comment to retain as a reference.